source/calendar/calendar_api.cpp

```diff
@@ -35,60 +35,58 @@ www.navitia.io
 
 namespace navitia { namespace calendar {
 
-pbnavitia::Response calendars(const navitia::type::Data &d,
-                              const boost::posix_time::ptime& current_datetime,
```

Comment:
current_dt is in pb_convertor, and is used only in the function for pb_creator creation.